openadapt/record.py

```diff
@@ -1205,6 +1232,10 @@ def record(
     if terminate_recording is not None:
         terminate_recording.set()
 
+    # TODO: rename on_ready
+    # TODO: consolidate terminate_recording and on_ready
```

Comment:
@KIRA009 can you please comment on this? Happy to jump on a call to clarify if it's easier.